src/main.rs

```diff
@@ -95,7 +98,14 @@ async fn main() -> Result<()> {
             run_serve(source.unwrap_or_else(|| ".".into()), port).await?;
         }
         Commands::New { name } => new_zine_project(name)?,
+        Commands::Version => {
+            let version = env!("ZINE_VERSION");
```

Comment:
I think here we can call `option_env!("CARGO_PKG_VERSION")` directly to get the version, no need to rely on `cargo:rustc-env`.